config/config.py

```diff
@@ -12,6 +12,14 @@ class EndoLightConfig:
     scaling_factor: float = 1
     euler_rotation: List[float] = field(default_factory=lambda: [0, 0, 0])
 
+@dataclass
+class TumorParticleConfig:
+    stl_file: str = MISSING
+    density: float = 4
+    volume_max: float = 0.1
+    scaling_range: Tuple[float] = field(default=(0.1, 1))
```

Comment:
A list might actually be a better idea to use instead of a Tuple. That way the user can change one value if they want. Tuple values can't be changed after being created.